src/main/java/org/reactome/server/export/tasks/HumanPathwaysWithDiagrams.java

```diff
@@ -0,0 +1,130 @@
+package org.reactome.server.export.tasks;
+
+import org.reactome.server.export.annotations.DataExport;
+import org.reactome.server.export.tasks.common.DataExportAbstract;
+
+import java.io.IOException;
+import java.nio.file.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+// JSON parsing imports (Jackson, for example)
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Export Pathway Summation Mapping from Neo4j
+ */
+@DataExport
+public class HumanPathwaysWithDiagrams extends DataExportAbstract {
+
+    private static final String DIAGRAM_DIRECTORY = "/data/diagrams";
+
+    private static final Set<String> EXCLUDED_RENDERABLE_CLASSES = new HashSet<>(
+    Arrays.asList(
+        "EncapsulatedNode",
+        "ProcessNode"
+    )
+);
```

Comment:
It looks like the indentation is a bit off here - I think everything can be indented one level?